release/release_data_tests.yaml

```diff
@@ -366,14 +367,23 @@
         - RAY_worker_killing_policy=retriable_lifo
       pip:
         - ray[default]
-    cluster_compute: all_to_all_compute.yaml
+    cluster_compute: dataset/all_to_all_compute.yaml
 
   run:
     timeout: 7200
     script: >
-      python sort_benchmark.py --num-partitions=1000 --partition-size=1e9 --shuffle
+      python dataset/sort_benchmark.py --num-partitions=1000 --partition-size=1e9 
+      --shuffle
 
+  variations:
+    - __suffix__: chaos
+      run:
+        prepare: > 
+          python setup_chaos.py --chaos TerminateEC2Instance --kill-interval 600 
+          --max-to-kill 2 
```

Comment:
More a question for my own knowledge, but do we guarantee any kills here or is this at most 2 will be killed?

Reply:
It depends on how long the job runs for. If the job finishes before the first kill interval (600s * random[0, 1]), nothing gets killed.

IIRC these tests run long enough that we guarantee two kills